Package.swift

```diff
@@ -1,11 +1,12 @@
-// swift-tools-version:5.8
+// swift-tools-version:6.0
 
 import PackageDescription
 
 let swiftSettings: [SwiftSetting] = [.enableExperimentalFeature("StrictConcurrency=complete")]
 
 let package = Package(
     name: "swift-aws-lambda-events",
+    platforms: [.macOS(.v15)],
```

Comment:
Why do you need to add the platform requirement?

Reply:
If it is needed for swift-testing you could add a `@available` at the top of each suite. 

Also you could just wrap all the tests in a `#if compiler(>=6.0)` to avoid requiring swift 6